webpack.common.js

```diff
@@ -23,5 +26,13 @@ module.exports = {
   },
   plugins: [
     new CleanWebpackPlugin(),
+    new HtmlWebpackPlugin({
+      appMountId: 'app',
+      template: HtmlWebpackTemplate,
+    }),
   ],
+  output: {
+    filename: 'bundle.js',
```

Comment:
Output needs to be here to make 'clean-webpack-plugin' have somewhere to point to for it to work in both `webpack.dev.js` and `webpack.prod.js`. At least, that's what I think is going on. It is also how it is setup in https://webpack.js.org/guides/production/ .